packages/styles/src/components/form-check.scss

```diff
@@ -155,7 +178,7 @@
       border: form-check.$form-check-input-border-width solid CanvasText;
 
       &:checked {
-        background-color: SelectedItem;
+        background-color: SelectedItem !important;
```

Comment:
Sorry to pick at this again, but the contrast of the ckecked radio button is low against the black background in this scenario. I believe we should aim for the colors the native elements have
![image](https://github.com/swisspost/design-system/assets/1659006/c31a51d8-71d0-4cdf-853e-c4a054e7e6c8)
vs.
![image](https://github.com/swisspost/design-system/assets/1659006/6450b3f9-00f0-44cd-89fa-c0de2739e1b8)

Reply:
In my opinion, we should not match native elements. In the past, it has proven that it has some flaws, for example with focus style. 

I used the `SelectedItem` which, [according to W3C](https://www.w3.org/TR/css-color-4/#css-system-colors) can be used as a background for checkbox:

![image](https://github.com/swisspost/design-system/assets/12294151/32794020-359f-4f74-8760-4f7a5638faaa)

In Firefox, the color is almost the same as the `Highlight` (if you look at the W3C list, you'll notice the slight difference as well). You might think this is only because it's not supported, but it seems that it's not the case [according to the tests of the W3C](https://wpt.fyi/results/css/css-color/system-color-support.html?label=experimental&label=master&aligned).

### Browser comparison

On Blink browser (with focus on selected radio): 

![msedge_H0lYd3W5wI](https://github.com/swisspost/design-system/assets/12294151/ee117912-a5c1-4cce-9436-e0435d0ff167)

On Firefox (with focus on selected radio): 

![firefox_H4iMHLAx31](https://github.com/swisspost/design-system/assets/12294151/e610d504-01ce-4df2-8939-6c2fea905591)

So, in my opinion, we should let browsers/OS handle the color, but use the appropriate one from specification.

Reply:
This makes sense, although I'm not really happy to release something if we know it's not going to work for everyone.